tool/webqa-normalize-data.js

```diff
@@ -513,10 +516,15 @@ module.exports = {
             nargs: '+',
             help: 'Input JSON+LD files to normalize. Multiple input files will be merged in one.'
         });
+        parser.addArgument('--class-name-prefix', {
+            required: false,
+            help: 'The prefix of the class name, defaults to org.schema',
+            defaultValue: 'org.schema'
+        });
```

Comment:
Why is it the class name prefix and not just the class name?

Reply:
hmm... true

Reply:
addressed.